src/static/css/homepage.css

```diff
@@ -0,0 +1,26 @@
+.asd {
+    padding: 60px;
```

Comment:
how responsive is this padding on say, an iPhone7 where the width is 750 pixels? https://mediag.com/blog/popular-screen-resolutions-designing-for-all/